test/inspector/inspector-helper.js

```diff
@@ -0,0 +1,425 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const http = require('http');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const TIMEOUT = 15 * 1000;
+
+const mainScript = path.join(common.fixturesDir, 'loop.js');
+
+function send(socket, message, id, callback) {
+  const msg = JSON.parse(JSON.stringify(message)); // Clone!
+  msg['id'] = id;
+  const messageBuf = Buffer.from(JSON.stringify(msg));
+
+  const wsHeaderBuf = Buffer.allocUnsafe(16);
+  wsHeaderBuf.writeUInt8(0x81, 0);
+  let byte2 = 0x80;
+  const bodyLen = messageBuf.length;
+  let maskOffset = 2;
+  if (bodyLen < 126) {
+    byte2 = 0x80 + bodyLen;
+  } else if (bodyLen < 65536) {
+    byte2 = 0xFE;
+    wsHeaderBuf.writeUInt16BE(bodyLen, 2);
+    maskOffset = 4;
+  } else {
+    byte2 = 0xFF;
+    wsHeaderBuf.writeUInt32BE(bodyLen, 2);
+    wsHeaderBuf.writeUInt32BE(0, 6);
+    maskOffset = 10;
+  }
+  wsHeaderBuf.writeUInt8(byte2, 1);
+  wsHeaderBuf.writeUInt32BE(0x01020408, maskOffset);
+
+  for (let i = 0; i < messageBuf.length; i++)
+    messageBuf[i] = messageBuf[i] ^ (1 << (i % 4));
+  socket.write(
+      Buffer.concat([wsHeaderBuf.slice(0, maskOffset + 4), messageBuf]),
+      callback);
+}
+
+function processIncomingMessage(socket, buffer, handler) {
```

Comment:
Did you fix this? Either the new github review mechanism is lying to me, or socket is still a parameter here.